docs/source/conf.py

```diff
@@ -27,7 +31,7 @@
 exclude_patterns = []
 
 # Options for HTML output
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
 html_theme_options = {'fixed_sidebar': True, 'show_relbars': True}
```

Comment:
I don't think these options are supported by RTD theme.  Does `make html` issue any warning?

Reply:
yea there were warnings, I just removed the fixed sidebar and show_relbars option.

Reply:
Fixed sidebar and relbar were set up so that it's easier to navigate. Does RTD theme supports similar features?

Reply:
It does.  We still need to figure out the quicknav one though on the other hand.  Quick test showed that it disappeared )-: